### PR TITLE
Upgrade bourbon to 7.0

### PR DESCRIPTION
**Pull request by mgorabbani**
This PR upgrade `Bourbon` version  to 7. which caused code refactoring. 
`Bourbon` library removes browser compability codes and  suggested to use `autoprefixer`. 
it also upgrade babel related libraries.

### CODE REVIEW

**Comment by timgluz** [comment]
@mgorabbani is it ready for review? I see that CircleCI is failing; maybe i  will wait until tomorrow;

**Comment by mgorabbani** [comment]
If you see old PRs it never passed. And i will work on it to run the circle
ci. I will need your help on it. So you can review it. But i still not sure
how it gets deployed as it was failing before too.

File: .circleci/config.yml [hunk 1]
```diff
@@ -36,9 +36,9 @@ jobs:
           aws-access-key-id: AWS_ACCESS_KEY_ID
           aws-region: AWS_DEFAULT_REGION
           aws-secret-access-key: AWS_SECRET_ACCESS_KEY
-          from: build/latest
+          from: build/
           overwrite: false
-          to: "s3://ui.wundery.com/v1.0.13" # update version number here.
+          to: "s3://ui.wundery.com"
```
Comment (timgluz) [inline .circleci/config.yml:41]: @mgorabbani  Good job! I have no blockers here; You could merge it and continue with epic work

**Review by timgluz — APPROVED** [review]
:+1:

**Pull request merged**